streaming/index.js

```diff
@@ -1331,11 +1350,6 @@ const startServer = async () => {
 
       // Decrement the metrics for connected clients:
       connectedClients.labels({ type: 'websocket' }).dec();
-
-      // ensure garbage collection:
-      session.socket = null;
-      session.request = null;
-      session.subscriptions = {};
```

Comment:
Was this useless? Or did some change make it useless? If so, which?

Reply:
This was me being overtly cautious, but I think it's fine to drop (it creates a typescript mess otherwise)

Reply:
The long term goal is to not even need this Session object, and instead use a set of maps / lists for managing channel membership. Then anything we need should be on either the `Websocket` or `Eventstream` instance, as noted in https://github.com/mastodon/mastodon/compare/main...ThisIsMissEm:mastodon:experimental/custom-eventsource-and-websocket-classes?expand=1

The only reason we currently need the "request" is because the user information is decorated on to that, which I've since started to refactor to just taking a request and returning a promise of user info.